.github/workflows/update-theme-libraries.yaml

```diff
@@ -0,0 +1,28 @@
+name: update library theme versions
+on:
+  pull_request:
+    paths:
+      - web/themes/new_weather_theme/assets/**/*.js
+      - web/themes/new_weather_theme/assets/**/*.css
+
+jobs:
+  should-test:
+    name: check if we should run tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: was the library file updated?
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          set +e
+          git diff --exit-code origin/main web/themes/new_weather_theme/new_weather_theme.libraries.yml > /dev/null
+          if [ "$?" -ne 0 ]
+          then
+            gh pr comment ${{ github.event.number }} -b "This PR modifies theme Javascript or CSS assets but does not update the theme libraries file. Did you mean to update the appropriate version information in the libraries file?"
+          fi
```

Comment:
No idea if this will work. It has to go into main before it'll run. Will no doubt be loads of fun to troubleshoot.

Reply:
it should work even if the output goes to /dev/null, but I think you're testing if the file was unmodified, so I think it should be `-ne 1`. is there a reason to use -ne or could we use -eq to make it clearer?